dialer.go

```diff
@@ -12,6 +12,10 @@ import (
 	"github.com/getlantern/ema"
 )
 
+var (
+	normalLiveConns = 1
```

Comment:
This can be a constant. Might also be better named `maxLiveConns` or something like that.

Reply:
I originally named it as `minLiveConns` but wanted to emphasize that it's the number in normal cases. I agree it's not a good name, so will change to `minLiveConns` for symmetry. Thanks!

Reply:
Changed at 8dc3a0e06d2a4d138bbb71585824708f45904fd2